_docs/_latest/setup/manual-deployment.md

```diff
@@ -0,0 +1,162 @@
+installation process.
+
+To complete this guide, you will need in-depth knowledge of GCP. For example,
+you will need to know how to use the gcloud command-line tool and where to bind
+a service account to a VM in the GCP Console. This guide refers to the installer
+and deployment templates for specific details of the commands to use.
+
+---
+
+## Create a project
+
+Create a new project to host Forseti. Forseti is intended to run in its own
+dedicated project so that access to its highly privileged permissions can be
+controlled.  Assign a billing account to the project.
+
+### Enable APIs
+
+For information about the APIs you need to enable in the project that hosts
+Forseti, see [required APIs]({% link _docs/latest/required_apis.md %}).
+
+## Deploy a server VM
+
+### Create a Forseti server service sccount
+
+Create a server service account by running the following command:
+
+```
+gcloud iam service-accounts create forseti-server-gcp-####### --display-name forseti-server-gcp-#######
+```
+
+Where `#######` is a random alphanumeric unique identifier.
+
+### Assign roles
+
+For information about the roles you need to assign to the Forseti server
+service account, see 
+[the server service account]({% link _docs/latest/concepts/service-accounts.html#the-server-service-account %}).
+
+### Create a Forseti server VM instance
+
+* For information about the specifications needed for a Forseti server
+  VM instance, see the [Compute Engine deployment template](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py)
+* When you create the instance, set the service account of the instance to the service account we just created or 
+if you already created the instance, you can [update the service account](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#changeserviceaccountandscopes).
+
+### Install the Forseti Server
+
+To install the Forseti Server, follow the steps below:
+
+1. SSH into the server VM
+1. Run the following command to become an Ubuntu user:
+    ```bash
+    sudo su ubuntu
+    ```
+1. `git clone` from the [forseti repo](https://github.com/GoogleCloudPlatform/forseti-security), and check out the [latest release](https://github.com/GoogleCloudPlatform/forseti-security/releases) by their tags.
+    ```bash
+    git clone https://github.com/GoogleCloudPlatform/forseti-security.git
+ 
+    git checkout tags/<tag_number>
+    ``` 
+1. Follow the setup instructions in the [startup-script](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/deployment-templates/compute-engine/server/forseti-instance-server.py#L109) 
+to install Forseti server and all the necessary components like Fluentd and Cloud SQL Proxy.
+1. Create firewall rules
+    ```bash
+    # Note: You will need the email address of your service account, you can get the email address by running command
```

Comment:
Split this into two sentences:

`# To create firewall rules, you will need the email address of your service account. You can get the service account's email address by running gcloud iam service-accounts-list.`

Reply:
Updated